multiqc/multiqc.py

```diff
@@ -746,6 +751,13 @@ def __rich_measure__(self, console: rich.console.Console, options: rich.console.
         report.runtimes["mods"][run_module_names[mod_idx]] = time.time() - mod_starttime
     report.runtimes["total_mods"] = time.time() - total_mods_starttime
 
+    # Update report with software versions provided in configs
+    software_versions.update_versions_from_config(config, report)
+
+    # Add section for software versions if any are found
+    if not config.skip_versions_section and report.software_versions and len(report.modules_output) > 0:
```

Comment:
I'm not sure that we need to check that there is module output - why not let people generate a MultiQC report with _only_ software versions? (Presumably from `*_mqc_versions.yml` files?)

Reply:
Changed in 23f55a03